react-native/react/actions/tracker.js

```diff
@@ -165,38 +186,26 @@ function serverCallMap (dispatch: Dispatch): CallMap {
       return false
     },
     finish: (params: {sessionID: number}) => {
-
-
+      const username = sessionIDToUsername[params.sessionID]
       // Check if there were any errors in the proofs
-      dispatch({type: Constants.updateProofState})
+      dispatch({type: Constants.updateProofState, payload: {username}})
 
       dispatch({
         type: Constants.markActiveIdentifyUi,
         payload: {
-          active: false
+          active: false,
+          username
         }
       })
     }
   }
 
-  dispatch({
```

Comment:
well the flow is kinda different as at this point we don't have a username or substore. so i can do that logic in the start: or really maybe we should just have that logic in the reducer init since it'll always be the same value on start right?